### ISSUE TITLE
Req: Option to disable increasing fingerprint entropy

### ISSUE THREAD

**Issue by rugabunda** [cowlicks/privacypossum#188]
Ghacks explains this somewhat here, UA spoofing is best left to privacy.resistFingerprinting, the other features are fine however. Please offer an option to disable your fingerprinting system, it wrecks firefoxes inbuilt fingerprinting protection by making users more unique looking.

**Comment by rugabunda** [comment]
Via Ghacks: https://github.com/ghacksuserjs/ghacks-user.js/issues/395#issuecomment-377892740

You've spent a lot of time with containers etc, I've spent far too much time on FP'ing (5 or 6 years). Trust me, you do not need to waste resources on this. Solutions already exist, especially the RFP pref one, which can do things that extensions cannot, since its baked into the code. Not only that, but it has patched long standing holes in eg navigator (which only applies when RFP is enabled).

TL;DR: waste of time

    it's extremely complicated and takes a lot of work
    its already been done, a thousand times
    is already available in Firefox behind a pref
    or an aspect of it like canvas has special dedicated extensions
    deviation from RFP can easily render RFP useless (so your features may just screw things up )
        we even have an RFP alternative section in the user.js, all inactive, so users know these should be at default when using RFP
    takes away resources from the core functionality/purpose of TC"
